src/loops/simple_programs_with_repitition/hard/q5/NowThatsPerfect.java

```diff
@@ -0,0 +1,36 @@
+package loops.simple_programs_with_repitition.hard.q5;
+
+/*
+Write a program with the class name "NowThatsPerfect" to accept a number from the user and determine whether it is a "Perfect" number.
+If it is a perfect number print: "x is a Perfect Number!". A perfect number is one whose factors sum up to equal the given number.
```

Comment:
Even though it is in the sample output, I think what it should print if it is not a perfect number should also be specified in the question text.